src/main/app.ts

```diff
@@ -25,24 +26,29 @@ const APP_INSIGHTS_KEY = config.secrets ? config.secrets["em-icp"]["AppInsightsI
 
 const logger = Logger.getLogger("app");
 
-const tlsOptions = {
-  password: REDIS_PASSWORD,
-  tls: true,
-};
-const redisOptions = config.redis.useTLS === "true" ? tlsOptions : {};
-export const redisClient = new Redis(config.redis.port, config.redis.host, redisOptions);
-
-redisClient.on("ready", () => {
-  console.log("Redis is ready");
-});
-
-redisClient.on("error", (err: { message: string; }) => {
-  console.log("Error in Redis: ", err.message);
-});
-
 export const app = express();
 app.locals.ENV = env;
 
+export let redisClient;
+if (env === "test") {
+  redisClient = new redisMock(config.redis.port, config.redis.host);
```

Comment:
not sure about this, this feels like test logic leaking into the app
is there a way to do with environment files?